File: 0657-robot-return-to-origin/0657-robot-return-to-origin.java
class Solution {
    public boolean judgeCircle(String moves) {
        int count=1;
        int c=0;
        for(int i=0;i<moves.length();i++){
            if(moves.charAt(i)=='R'){
                c++;
            }
            else if(moves.charAt(i)=='L'){
                c--;
            }
            else if(moves.charAt(i)=='U'){
                count++;
            }
            else if(moves.charAt(i)=='D'){
                count--;
            }
        }
        if(count==1&&c==0){
            return true;
        }
        else{
            return false;
        }
    }
}

File: 0657-robot-return-to-origin/0657-robot-return-to-origin.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        count1=0
        count2=0
        for i in moves:
            if i=='R':
                count1+=1
            elif i=='L':
                count1-=1
            elif i=='U':
                count2+=1
            elif i=='D':
                count2-=1
        if count1==0 and count2==0:
            return True
        else:
            return False